doc/man1/openssl-verification-options.pod

```diff
@@ -267,9 +375,10 @@ keys shorter than 1024 bits.
 
 =item B<-partial_chain>
 
-Allow verification to succeed even if a I<complete> chain cannot be built to a
-self-signed trust-anchor, provided it is possible to construct a chain to a
-trusted certificate that might not be self-signed.
+Allow verification to succeed also if an incomplete chain can be built,
```

Comment:
d/also/
s/,/./ and capatilise the next sentence.

Reply:
Done, replacing "i.e." by "That is".